cordformation/src/test/kotlin/net/corda/plugins/NetworkParameterOverridesTest.kt

```diff
@@ -0,0 +1,58 @@
+package net.corda.plugins
+
+import com.typesafe.config.*
+import org.assertj.core.api.Assertions.assertThat
+import org.gradle.api.Project
+import org.gradle.testfixtures.ProjectBuilder
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+
+class NetworkParameterOverridesTest {
+
+    @Before
+    fun setUp() {
+    }
+
+    @After
+    fun tearDown() {
+    }
+
+    @Test
+    fun toConfig() {
+        val project = ProjectBuilder.builder().build()
+        val networkParameterOverrides = createNetworkParameterOverrides(project)
+
+        val config = networkParameterOverrides.toConfig()!!
```

Comment:
This cannot be `null`, so you don't need the `!!` operator here.

Reply:
Fixed 06e93caf3c11a1d1585888fe9b996319900595bc